src/components/validations/validation-icon.style.js

```diff
@@ -19,6 +19,10 @@ const ValidationIconStyle = styled.div`
     color: ${({ validationType, theme }) => theme.colors[validationType]};
   }
 
+  ${StyledIcon}:focus {
+    outline: ${({ theme }) => (isClassic(theme) ? 'none' : `2px solid ${theme.colors.focus}`)}
```

Comment:
Can we move it to the `isClassic` condition below? So we can have all in one. Also you can extract `isClassic` condition to an external file.